config/database.php

```diff
@@ -64,7 +64,7 @@
             'charset' => 'utf8',
             'prefix' => '',
             'schema' => 'public',
-            'sslmode' => 'prefer',
+            'sslmode' => 'disable',
```

Comment:
Is this change really necessary for running unit tests?

Reply:
Actually I run tests with PostgreSQL and running in local mode it is necessary to change that, I will improve that to use ENV_VARS.